lib/ui/menu/settings_drawer/customization_menu_view.dart

```diff
@@ -75,10 +75,7 @@ class CustomizationMenuView extends ConsumerWidget {
                       _SettingsListItem.withDefaultValue(
                         heading: localizations.primaryCurrency,
                         defaultMethod: primaryCurrency,
-                        icon: primaryCurrency.primaryCurrency ==
-                                AvailablePrimaryCurrencyEnum.fiat
-                            ? UiIcons.primary_currency
-                            : UiIcons.primary_currency_uco,
+                        icon: Icons.currency_exchange_outlined,
```

Comment:
Not managed anymore because the state of the selected item is not managed by the other entries neither